content/reference/components@es.md

```diff
@@ -1,45 +1,50 @@
 ---
-$title: Componentes / Etiquetas
+$title: Componentes
 $order: 0
 toc: true
 ---
```

Comment:
Please remove the added frontmatter from all the files. Only the translated $title should be present on translated files. Thank you.